test/e2e/specs/form-interaction-tests.js

```diff
@@ -50,18 +50,21 @@ module.exports = {
 
     browser.end()
   },
+
   'Toggle show optional fields': function (browser) {
     browser.click('#toggle-btn')
     browser.expect.element('#string').to.be.not.visible
     browser.expect.element('#show-fields-icon').to.be.visible
-    browser.assert.cssClassPresent('#show-fields-icon', 'fa-eye')
+    browser.expect.element('#show-fields-icon').to.have.attribute('class').which.contains('fa-eye')
     browser.click('#toggle-btn')
     browser.expect.element('#string').to.be.visible
-    browser.assert.cssClassPresent('#show-fields-icon', 'fa-eye-slash')
+    browser.expect.element('#show-fields-icon').to.have.attribute('class').which.contains('fa-eye-slash')
     browser.end()
   },
-  'Check if asterix class is present on integer element': function (browser) {
-    browser.assert.cssClassPresent('#integer-fs', 'required-field')
+
+  'Check if asterisk class is present on integer element': function (browser) {
+    browser.expect.element('#integer-fs').to.have.attribute('class').which.contains('required-field')
+    browser.expect.element('#integer-fs label').to.have.attribute('class').which.contains('field-label')
     browser.end()
   }
 }
```

Comment:
Add test that reproduces the bug, or in this case shows that the bug is not there anymore? I can do this myself since I have created the issue. Shall I checkout the branch?

Reply:
No dont, see #64

Reply:
And I could not figure out how to check for ::after content inside the e2e element selector